### ISSUE TITLE
When exporting a polygon, the export crashes (2) 

### ISSUE THREAD

**Issue by SaturnHafen** [simulate-digital-rail/demo#7]
Hi again from the bachelors project :)

When exporting an area, the export crashes. (The area in question: `52.288322586002984 12.808685302734377 52.16803594836457 12.992706298828127 52.26311463698562 13.064117431640627`)

The issue seems to be that the osm data contain a question mark (`?`) in a max-speed field.

Server backtrace:
```
127.0.0.1 - - [19/Jan/2023 10:59:15] "GET /run?polygon=52.47608904123904+9.530639648437502+52.221069523572794+9.503173828125+52.251346020673665+9.975585937500002+52.52958999943304+10.030517578125002&mode=planpro HTTP/1.1" 500 -
Error on request:
Traceback (most recent call last):
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/werkzeug/serving.py", line 335, in run_wsgi
    execute(self.server.app)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/werkzeug/serving.py", line 322, in execute
    application_iter = app(environ, start_response)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2548, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.handle_exception(e)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/saturnhafen/Code/Bachelorprojekt/yaramo-testing/demo/app.py", line 19, in run_converter
    topology = ORMImporter().run(polygon)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/importer.py", line 136, in run
    self._add_signals(path, current_edge, node, next_top_node)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/importer.py", line 90, in _add_signals
    signal.additional_signals = get_additional_signals(node)
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/utils.py", line 190, in get_additional_signals
    [AdditionalSignalZs3.AdditionalSignalSymbolZs3(0 if s == "off" else int(s)/10) for s in get_zs_values(signal.tags, "railway:signal:speed_limit:speed")])
  File "/home/saturnhafen/.cache/pypoetry/virtualenvs/demo-SE8fofqi-py3.10/lib/python3.10/site-packages/orm_importer/utils.py", line 190, in <listcomp>
    [AdditionalSignalZs3.AdditionalSignalSymbolZs3(0 if s == "off" else int(s)/10) for s in get_zs_values(signal.tags, "railway:signal:speed_limit:speed")])
ValueError: invalid literal for int() with base 10: '?'
```

**Comment by jeriox** [comment]
hi again :)

this should be fixed as of https://github.com/simulate-digital-rail/orm-importer/commit/f26e3ccf344cf530959588924df540de5cd406fd#diff-0efd7fe17c9656a7c638d57a2a613cfb37a7c3951ce2106ac3d31975ebecba19R171 as well, did you try with the latest version?

**Comment by ctiedt** [comment]
Should be fixed by #5. Please re-open this issue if the problems persist.
